app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -1,457 +1,539 @@
-    <string name="did_you_mean">您是不是要找： %1$s ?</string>
-    <string name="share_dialog_title">分享视频</string>
+    <string name="share_dialog_title">分享给...</string>
     <string name="choose_browser">选择浏览器</string>
-    <string name="screen_rotation">旋转</string>
-    <string name="use_external_video_player_title">使用外置视频播放器</string>
-    <string name="use_external_audio_player_title">使用外置音频播放器</string>
     <string name="download_path_title">视频下载路径</string>
-    <string name="download_path_summary">下载视频的存储路径</string>
-    <string name="download_path_dialog_title">输入视频存储路径</string>
-    <string name="default_resolution_title">默认分辨率</string>
-    <string name="play_with_kodi_title">用Kodi播放</string>
-    <string name="kore_not_found">找不到Kore，您要安装Kore吗？</string>
-    <string name="show_play_with_kodi_title">显示“用Kodi播放”的选项</string>
-    <string name="show_play_with_kodi_summary">显示 Kodi 媒体中心播放视频的选项</string>
-    <string name="play_audio">音频</string>
-    <string name="default_audio_format_title">默认音频格式</string>
+    <string name="download_path_summary">已下载的视频存储在这里</string>
+    <string name="download_path_dialog_title">请选择下载视频的保存位置</string>
+    <string name="download_path_audio_summary">已下载的音频存储在这里</string>
+    <string name="download_path_audio_dialog_title">选择下载音频的储存位置</string>
+    <string name="autoplay_by_calling_app_title">自动播放</string>
+    <string name="play_with_kodi_title">使用Kodi播放</string>
     <string name="theme_title">主题</string>
-    <string name="dark_theme_title">灰暗</string>
-    <string name="light_theme_title">明亮</string>
-
+    <string name="light_theme_title">浅色</string>
+    <string name="dark_theme_title">暗黑</string>
+    <string name="black_theme_title">黑色</string>
     <string name="download_dialog_title">下载</string>
-    <string name="next_video_title">即将播放</string>
-    <string name="show_next_and_similar_title">显示下一部和相似的视频</string>
-    <string name="url_not_supported_toast">不支援此网址</string>
-    <string name="content_language_title">默认内容语言</string>
-    <string name="settings_category_video_audio_title">视频和音频</string>
+    <string name="next_video_title">下一个</string>
+    <string name="url_not_supported_toast">不支持的 URL</string>
     <string name="settings_category_appearance_title">外观</string>
     <string name="settings_category_other_title">其他</string>
-    <string name="background_player_playing_toast">后台播放</string>
-    <string name="play_btn_text">播放</string>
+    <string name="all">全部</string>
+    <string name="channel">频道</string>
+    <string name="yes">是</string>
+    <string name="later">稍后</string>
     <string name="network_error">网络错误</string>
-
-    <string name="list_thumbnail_view_description">视频预览缩略图</string>
-    <string name="detail_thumbnail_view_description">视频预览缩略图</string>
-    <string name="detail_uploader_thumbnail_view_description">上传者的头像缩图</string>
+    <plurals name="videos">
+        <item quantity="one">视频</item>
+        <item quantity="other">视频</item>
+    </plurals>
```

Comment:
Well yes, but actually no.
It has been two month since %s is removed on Weblate, https://hosted.weblate.org/translate/newpipe/strings/en/?q=videos&offset=4, and some other languages too, including french, and for using french almost everyday I can tell you it's not crashing. We should know if %s or not should be used, and edit all the files with this string to match or not this change.

Reply:
Yes,it wouldn't crash,but I will update as well.

Reply:
Ah, yes. it is not crashing. however, the number is not displayed. These strings are used for example when displaying info about a channel. It says either "1 video" or "123 videos". So we need a `%s` in the translation, because the number is relevant info for the user. But We cannot simply insert `%s` to all plurals, because we do not know where to place it.

Reply:
It just print the original English strings. Anyway, now I understand why some strings are still English in my own build, thanks.

Reply:
> Ah, yes. it is not crashing. however, the number is not displayed. These strings are used for example when displaying info about a channel. It says either "1 video" or "123 videos". So we need a `%s` in the translation, because the number is relevant info for the user. But We cannot simply insert `%s` to all plurals, because we do not know where to place it.

It's used in the playlist. I edited english strings through weblate with %s, and it triggered the edit with all languages (you have to confirm if it matches the source), so people will see and update

Reply:
Great, but do you know, the  Weblate source file of Simplified Chinese is not this file (zh-cn),is b+zh-hans . I opened this pr is to move b+zh-hans to this,and then ask maintainer to change source and remove b+zh-hans.